pkg/network/usm/utils/file_registry.go

```diff
@@ -122,14 +122,7 @@ var (
 
 // getSamplePath returns a sample path for a given path without the prefix /usr.
 func getSamplePath(path string) string {
-	index := strings.Index(path, "/usr")
-	samplePath := path
-
-	if index != -1 {
-		samplePath = path[index+len("/usr"):]
-	}
-
-	return samplePath
+	return strings.TrimPrefix(path, "/proc")
```

Comment:
as we discussed by heart
let's understand how much this optimization saves before adding any cutoff

Reply:
also, documentation is wrong
you should also explain why you're removing the prefix